### ISSUE TITLE
trivy k8s all only scan context namespace

### ISSUE THREAD

**Issue by senemamourr** [aquasecurity/trivy#3804]
when i do trivy k8s all  --compliance k8s-cis --report all --format table --severity HIGH,CRITICAL

it scan only the namespace defined in the context of .kube/config not all namespace on the cluster.
how to tell it to scan all the namespace ?

**Comment by chen-keinan** [comment]
@senemamourr have you tried `trivy k8s cluster ...`

**Comment by senemamourr** [comment]
yes, but it create a job scan which need to access on hostPath  which is not allowed by the psa.
I had to use an older version to run the compliance.

**Comment by chen-keinan** [comment]
> yes, but it create a job scan which need to access on hostPath which is not allowed by the psa. I had to use an older version to run the compliance.

@senemamourr could you please try with latest trivy , it include improvement with `hostPath` 
if you are getting any error , could you please put it here ?
